app/assets/javascripts/discourse/app/deprecation-workflow.js

```diff
@@ -0,0 +1,6 @@
+import setupDeprecationWorkflow from "ember-cli-deprecation-workflow";
+import DEPRECATION_WORKFLOW from "discourse-common/deprecation-workflow";
+
+// We're using RAISE_ON_DEPRECATION in environment.js instead of
+// `throwOnUnhandled` here since it is easier to toggle.
```

Comment:
Is this comment still relevant?

Reply:
More than ever 😄 Previously it wasn't 100% correct because we used both methods, but now sine it's not possible to change throwOnUnhandled option at runtime we rely on our custom logic based on the other flag.